packages/types/src/interfaces.ts

```diff
@@ -24,30 +24,32 @@ export interface Store {
 
 export interface CosmosBlock {
```

Comment:
Thinking about this some more, we can revert these changes. What we had before is more useful.

But i think we need to include more in the block, we should include the block results in here too.